qa/tasks/vstart_runner.py

```diff
@@ -773,6 +766,23 @@ def list_connections():
 
         self.gather_mount_info()
 
+    def _set_fuse_daemon_pid(self):
+        # NOTE: When a command <args> is launched with sudo, two processes are
+        # launched, one with sudo in <args> and other without. Make sure we
+        # get the PID of latter one.
+        for i in range(5):
+            time.sleep(5)
+            try:
+                sock = self.find_admin_socket()
+            except RuntimeError:
+                pass
+            pid = re.search(r'client\.\d+\.(\d+)\.asok$', sock).group(1)
+            if pid:
```

Comment:
Where the `pid` will be used ? I think this is why my test will always fail.

Reply:
You should also fix this in fuse_mount.py too.

Reply:
Fixed.

Reply:
> You should also fix this in fuse_mount.py.

Skipping this based on my chat with Xiubo. The rationale is that we don't need this change since jobs in integration tests kill ceph-fuse properly already unlike locally.